src/utils/viewer/viewer.cpp

```diff
@@ -273,6 +273,12 @@ Viewer::Viewer(const Arguments& arguments)
       .setHelp("object-dir",
                "Provide a directory to search for object config files "
                "(relative to habitat-sim directory).")
+      .addBooleanOption("use-ortho-camera")
+      .setHelp("use-ortho-camera", "Use orthographic camera to view scene.")
+      .addOption("ortho-scale", ".1")
```

Comment:
You could combine these two args, but making use-ortho-camera True if orthoscale != 0/ -1/ NaN. etc... or even if it's just specified at all.

Reply:
I may not agree with you.
Compared to the  "use-ortho-camera", "ortho-scale" is a more advanced setting. You cannot expect any user is willing to set it. Most of them simply needs a toggle while keeping the other default parameters not changed.